src/main.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { createServer } = require('./createServer');
+const { createServer } = require('./createServer').default;
```

Comment:
The import statement for `createServer` is incorrect. The `.default` property should not be used here. Change `require('./createServer').default` to `require('./createServer')` to correctly import the `createServer` function.